src/app/[locale]/_component/feedbackMessage/feedbackMessage.tsx

```diff
@@ -79,7 +90,12 @@ export function FeedbackMessage({
                 sx={{
                   marginLeft: 2,
                 }}
-                onClick={() => pushWithLocale(secondButton.href || '')}
+                onClick={() => {
+                  pushWithLocale(secondButton.href || '');
+                  if (secondButton.onClick) {
+                    secondButton.onClick();
+                  }
+                }}
```

Comment:
as above

Reply:
Yes, I introduced it to have a callback for those buttons in the component to handle the call back events, as each page has different trackevent